packages/react-storage/src/components/StorageBrowser/Views/Controls/Navigate.tsx

```diff
@@ -114,15 +110,20 @@ export function NavigateControl(): React.JSX.Element {
 
         const isCurrent = index === history.length - 1;
 
-        return (
+        return isCurrent ? (
+          <ListItemElement
+            className={`${BLOCK_NAME}__item`}
+            aria-current="page"
+          >
+            <SpanElement variant="navigate-current">{displayValue}</SpanElement>
+          </ListItemElement>
+        ) : (
```

Comment:
This complicates the logic needed for having a single render item, the conditional logic for rendering plain text needs to be encapsulated to the `NavigateItem`

Reply:
That makes sense. In that case, I updated NavigateItem to just have the prop "isCurrent", the it uses that to render out the correct markup/Separator.